.github/workflows/build.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Setup snyk
         uses: snyk/actions/setup@0.3.0
       - name: Snyk test
-        run: snyk test -d --all-sub-projects --org=hypertrace --severity-threshold=low --configuration-matching="^runtimeClasspath$"
+        run: snyk test -d --all-sub-projects --org=hypertrace --severity-threshold=low --policy-path=.snyk --configuration-matching="^runtimeClasspath$"
```

Comment:
maybe it doesn't work since it's looking at `--configuration-matching="^runtimeClasspath$"`? Is there a runtimeClasspath in the agent?

Reply:
Actually jk. See if it runs locally.